erpc/networks.go

```diff
@@ -186,10 +186,17 @@ func (n *Network) Forward(ctx context.Context, req *upstream.NormalizedRequest)
 	resp, execErr := n.failsafeExecutor.
 		WithContext(ctx).
 		GetWithExecution(func(exec failsafe.Execution[common.NormalizedResponse]) (common.NormalizedResponse, error) {
+			lg.Debug().Msg("Entered GetWithExecution")
+
 			n.upstreamsMutex.RLock()
 			upsList := n.Upstreams
 			n.upstreamsMutex.RUnlock()
 
+			if upsList == nil {
+				lg.Error().Msg("upsList is nil")
+				return nil, errors.New("upsList is nil")
+			}
```

Comment:
This shouldn't happen here, because u.Upstreams must never be null at this stage. We can have this check during network initialization.